src/installer/index.ts

```diff
@@ -35,16 +25,11 @@ export async function installRuby(
   rubyVersionText: string,
   rubyPackageVersion: string | null
 ) {
-  const rubyVersionWithoutPatchVersion = removePatchVersion(rubyVersionText);
-  if (!validateRubyVersionText(rubyVersionWithoutPatchVersion)) {
-    throw new Error(`Invalid version text: ${rubyVersionText}`);
-  }
-
```

Comment:
These process is moved to package_version_info.ts because `rubyPackageVersion` needs teeny versions.